components/eam/bld/configure

```diff
@@ -323,6 +324,7 @@ my %opts = (
 GetOptions(
     "age_of_air_trcs!"          => \$opts{'age_of_air_trcs'},
     "cldera_sai_trcs!"          => \$opts{'cldera_sai_trcs'},
+    "cldera_passive_trcs!"        => \$opts{'cldera_passive_trcs'},
```

Comment:
It would be good to line up the `=>` indenting with the lines above and below.

Reply:
done

Reply:
done